lib/Doctrine/Migrations/InlineParameterFormatter.php

```diff
@@ -53,7 +55,7 @@ public function formatParameters(array $params, array $types): string
         return sprintf('with parameters (%s)', implode(', ', $formattedParameters));
     }
 
-    private function formatParameter(mixed $value, string|int $type): string|int|bool|float|null
+    private function formatParameter(mixed $value, string|int|ArrayParameterType|ParameterType $type): string|int|bool|float|null
```

Comment:
I wonder if we should change the type to `mixed`. After all, the method discards any value that is not a string anyway.

Reply:
@derrabus pushed - if we ok with mixed, then I can even change back the bump in composer.json, but then we would not have a test case for ArrayParameterType as it does not exist in DBAL 3.5